core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/RawlsBillingProjectComponent.scala

```diff
@@ -350,6 +350,16 @@ trait RawlsBillingProjectComponent {
         .result
         .map(_.headOption.map(RawlsBillingProjectRecord.toBillingProjectSpendExport))
 
+    def getBillingProjectsSpendConfiguration(
+      billingProjectNames: Seq[RawlsBillingProjectName]
+    ): ReadAction[Seq[Option[BillingProjectSpendExport]]] =
+      rawlsBillingProjectQuery
+        .withProjectNames(billingProjectNames)
+        .result
+        .map(projectRecords =>
+          projectRecords.map(record => Some(RawlsBillingProjectRecord.toBillingProjectSpendExport(record)))
+        )
+
```

Comment:
is it important for this to return `BillingProjectSpendExport` wrapped inside an `Option`? It doesn't look like there's anything optional; it always uses `Some`. You might be able to simplify this to:
```scala
    def getBillingProjectsSpendConfiguration(
      billingProjectNames: Seq[RawlsBillingProjectName]
    ): ReadAction[Seq[BillingProjectSpendExport]] =
      rawlsBillingProjectQuery
        .withProjectNames(billingProjectNames)
        .result
        .map(projectRecords => projectRecords.map(RawlsBillingProjectRecord.toBillingProjectSpendExport))
```